CONTRIBUTING.md

```diff
@@ -0,0 +1,15 @@
+### Submitting a new package
```

Comment:
This `CONTRIBUTING.md` should not be here. You have based your submission off the `master` branch rather than the `new-pr` branch.